lib/ex_unit/lib/ex_unit/runner_stats.ex

```diff
@@ -25,6 +25,11 @@ defmodule ExUnit.RunnerStats do
     {:noreply, %{map | total: total + 1, skipped: skipped + 1}}
   end
 
+  def handle_cast({:case_finished, %ExUnit.TestCase{state: {:failed, _reason}}}, %{failures: failures, totals:
+  totals} = map) do
+    {:noreply, %{map | failures: failures + 1, totals: totals + 1}}
```

Comment:
Maybe it would be better to compute the amount of tests in the case? Something like `length(case.__ex_unit__(:tests))` and add that instead of 1?